server/controllers/temporary-accommodation/manage/bookingsController.test.ts

```diff
@@ -30,12 +29,21 @@ import {
   insertBespokeError,
   insertGenericError,
 } from '../../../utils/validation'
+import { isApplyEnabledForUser } from '../../../utils/userUtils'
 import BookingsController from './bookingsController'
 
 jest.mock('../../../utils/bookingUtils')
 jest.mock('../../../utils/restUtils')
 jest.mock('../../../utils/validation')
 jest.mock('../../../utils/utils')
+jest.mock('../../../utils/userUtils', () => {
+  const module = jest.requireActual('../../../utils/userUtils')
+
+  return {
+    ...module,
+    isApplyEnabledForUser: jest.fn(),
+  }
```

Comment:
I wouldn't worry about this tbh, think just jest.mock('../../../utils/userUtils') is fine

There's a few odd cases in the codebase where we need a mocked version of a function and a real version in the same test file, but those are the only places we really use this pattern

Reply:
:recycle: